rubygem-smart_proxy_openscap/rubygem-smart_proxy_openscap.spec

```diff
@@ -0,0 +1,101 @@
+%global gem_name smart_proxy_openscap
+
+%global foreman_proxy_bundlerd_dir /usr/share/foreman-proxy/bundler.d
+%global foreman_proxy_pluginconf_dir /etc/foreman-proxy/settings.d
+%global spool_dir /var/spool/foreman-proxy/openscap
+%global proxy_user foreman-proxy
+
+Name: rubygem-%{gem_name}
+Version: 0.3.0
+Release: 1%{?dist}
+Summary: OpenSCAP plug-in for Foreman's smart-proxy.
+Group: Applications/Internet
+License: GPLv2+
+URL: http://github.com/openscap/smart_proxy_openscap
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+#Requires: ruby(release)
+Requires: ruby(rubygems)
+Requires: foreman-proxy >= 1.7.0-0.develop.201410221520
+Requires: crontabs
+#BuildRequires: ruby(release)
+BuildRequires: rubygems-devel
+BuildRequires: ruby
+BuildArch: noarch
+Provides: rubygem(%{gem_name}) = %{version}
+Obsoletes: rubygem-foreman-proxy_openscap
+
+%description
+A plug-in to the Foreman's smart-proxy which receives bzip2ed ARF files
+and forwards them to the Foreman.
+
+%prep
+gem unpack %{SOURCE0}
+%setup -q -D -T -n  %{gem_name}-%{version}
+gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+
+%build
+# Create the gem as gem install only works on a gem file
+gem build %{gem_name}.gemspec
+
+# %%gem_install compiles any C extensions and installs the gem into ./%gem_dir
+# by default, so that we can move it into the buildroot in %%install
+%gem_install
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+       %{buildroot}%{gem_dir}/
+mv %{buildroot}%{gem_instdir}/%{gem_name}.gemspec %{buildroot}/%{gem_spec}
```

Comment:
I don't think this is right, it should be using the gem_spec generated by the gem build above, not the original.